rosdep/python.yaml

```diff
@@ -2656,6 +2656,22 @@ python-pyramid:
   fedora: [python-pyramid]
   gentoo: [dev-python/pyramid]
   ubuntu: [python-pyramid]
+python-pyshark-pip:
+  debian:
+    pip:
+      packages: [pyshark]
+  fedora:
+    pip:
+      packages: [pyshark]
+  gentoo:
+    pip:
+      packages: [pyshark]
+  ubuntu:
+    pip:
+      packages: [pyshark]
```

Comment:
Since these are pip wrappers for the underlying too. I suspect that they need tshark installed to be useful. If so these entries should have a depend key as well onto tshark.